src/python/pants/notes/2.1.x.rst

```diff
@@ -0,0 +1,92 @@
+2.1.x Stable Releases
+=====================
+
+This document describes releases leading up to the ``2.1.x`` ``stable`` series.
+
+See https://www.pantsbuild.org/v2.0/docs/release-notes-2-1 for an overview of the changes in this release, and
+
+2.1.0.dev0 (10/30/2020)
+----------------------
+
+User API Changes
+~~~~~~~~~~~~~~~~
+
+* Remove deprecated `--{fmt,lint}-per-target-caching` in favor of `--per-file-caching` (#11062)
+  `PR #11062 <https://github.com/pantsbuild/pants/pull/11062>`_
+
+* Remove post_stats from RunTracker (#11068)
+  `PR #11068 <https://github.com/pantsbuild/pants/pull/11068>`_
+
+* Remove stats_version in runtracker (#11065)
+  `PR #11065 <https://github.com/pantsbuild/pants/pull/11065>`_
+
+* Remove deprecated `--lang` and `--lc-all` options in favor of `--env-vars` (#11066)
+  `PR #11066 <https://github.com/pantsbuild/pants/pull/11066>`_
+
+* Add `:my_func` shorthand for the `entry_point` field (#11064)
+  `PR #11064 <https://github.com/pantsbuild/pants/pull/11064>`_
+
+* Remove deprecated `--protoc-runtime-targets` option (#11083)
+  `PR #11083 <https://github.com/pantsbuild/pants/pull/11083>`_
+
+* Remove deprecated `--interpreter-constraints` option for some Python tools (#11082)
+  `PR #11082 <https://github.com/pantsbuild/pants/pull/11082>`_
+
+* Remove deprecated `runtime_binary_dependencies` field (#11079)
+  `PR #11079 <https://github.com/pantsbuild/pants/pull/11079>`_
+
+* Remove deprecated `--spec-file` option in favor of `--spec-files` (#11076)
+  `PR #11076 <https://github.com/pantsbuild/pants/pull/11076>`_
+
+* Remove deprecated `[basic-auth]`, `[reporter]`, and `[run-tracker]` options (#11078)
+  `PR #11078 <https://github.com/pantsbuild/pants/pull/11078>`_
+
+* Deprecate the `compatibility` field in favor of `interpreter_constraints` (#11074)
+  `PR #11074 <https://github.com/pantsbuild/pants/pull/11074>`_
+
+* Remove deprecated `--print-exception-stacktrace` in favor of `--print-stacktrace` (#11085)
+  `PR #11085 <https://github.com/pantsbuild/pants/pull/11085>`_
+
+* Remove unused global options (#11077)
+  `PR #11077 <https://github.com/pantsbuild/pants/pull/11077>`_
+
+* Remove deprecated `python_requirement` BUILD file symbol (#11055)
+  `PR #11055 <https://github.com/pantsbuild/pants/pull/11055>`_
+
+Plugin API Changes
+~~~~~~~~~~~~~~~~~~
+
+* Deprecate the `platforms` field allowing a bare string (#11073)
+  `PR #11073 <https://github.com/pantsbuild/pants/pull/11073>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Restore EngineAware metadata() method with non-cpython-reliant data structures (#11054)
+  `PR #11054 <https://github.com/pantsbuild/pants/pull/11054>`_
+  `PR #11047 <https://github.com/pantsbuild/pants/pull/11047>`_
```

Comment:
This seems like an internal only change. Only Toolchain is consuming this API, as far as we know.